docs/howto/windows.txt

```diff
@@ -21,13 +23,19 @@ machine. At the time of writing, Python 3.7 is the latest version.
 
 To install Python on your machine go to https://python.org/downloads/. The
 website should offer you a download button for the latest Python version.
-Download the executable installer and run it. Check the box next to ``Add
-Python 3.7 to PATH`` and then click ``Install Now``.
+Download the executable installer and run it. Check the boxes next to ``Install
+launcher for all users (recommended)`` and ``Add Python 3.7 to PATH`` then
+click ``Install Now``.
 
 After installation, open the command prompt and check that the Python version
 matches the version you installed by executing::
 
-    python --version
+    ...\> py --version
+
+.. seealso::
+
+    The official `Using Python on Windows
+    <https://docs.python.org/3/using/windows.html>`_ document.
```

Comment:
This can use intersphinx to avoid hardcoding the full URL.

Reply:
Done. Thanks!